### ISSUE TITLE
Versioning of collections not appearing

### ISSUE THREAD

**Issue by alko-k** [surroundaustralia/VocPrez-theme-nvs#69]
Hi Nick,

could we please enable links related to a collection's version like http://nvs.surroundaustralia.com/collection/T01/1/ to appear in NVS? 

Thanks 
Alexandra

**Comment by nicholascar** [comment]
Assuming that the vocab you refer to has the URI <http://vocab.nerc.ac.uk/collection/T01/1/> then I can't yet do this as I cant get any info from the SPARQL endpoint for that vocab (see http://vocab.nerc.ac.uk/sparql/sparql?query=http%3A%2F%2Fvocab.nerc.ac.uk%2Fcollection%2FT01%2F1%2F)

**Comment by nicholascar** [comment]
I'm still going to need more instruction here: 

for T01, I can see that the Collection has this:

```turtle
<http://vocab.nerc.ac.uk/collection/T01/current/>
    ...
    owl:versionInfo "1" ;
    ...
.
```

So we can see that the Collection has the version "1" but do you want me to link to a version IRI like this:

```
<http://vocab.nerc.ac.uk/collection/T01/1/>
```

If so, how do I _know_ (from the data) what that IRI is _and_ how to I enable it? I can guess that such an IRI exists and make it (by using the version "1" to replace "current" in the Collection IRI) but it doesn't lead anywhere (there's no information about that IRI, see comment above).

So, if you want me to reveal a version-based IRI like `<http://vocab.nerc.ac.uk/collection/T01/1/>`, I think you might need to do two things:

1. specifically indicate the IRI
2. provide details for the IRI in the triplestore

For 1: Extend the version info like this:

```turtle
<http://vocab.nerc.ac.uk/collection/T01/current/>
    ...
    owl:versionInfo "1" ;
    owl:versionIRI <http://vocab.nerc.ac.uk/collection/T01/1/> ;
    ...
.
```

Then I'll know that such a version IRI exists and I can just present that as a click-able property for all Collections/Schemes etc.

For 2: this will be a back-end Oracle task I suppose! If I can see properties with the query `DESCRIBE <http://vocab.nerc.ac.uk/collection/T01/1/>` then I can make the API enable that link and show them

**Comment by nicholascar** [comment]
You could use `pav:hasCurrentVersion <http://vocab.nerc.ac.uk/collection/T01/1/> ;` instead of `owl:versionIRI <http://vocab.nerc.ac.uk/collection/T01/1/> ;` if you prefer, since you are already using `pav:hasCurrentVersion` at the Concept level. I mentioned `owl:versionIRI` above because it matches your current use of `owl:versionInfo` but the choice is yours!

**Comment by nicholascar** [comment]
Solved in addressing Issue #123
